common/persistence/class.PhpRedisDriver.php

```diff
@@ -103,33 +104,28 @@ private function connectToCluster(array $host, int $timeout, bool $persist)
     protected function callWithRetry($method, array $params, $attempt = 1)
     {
 
-        $success       = false;
+        $success = false;
         $lastException = null;
-        $result        = false;
+        $result = false;
 
-        $retry = $this->params['attempt'];
+        $retry = (int)$this->params['attempt'];
 
         while (!$success && $attempt <= $retry) {
             try {
-                $result = call_user_func_array([$this->connection , $method], $params);
+                $result = call_user_func_array([$this->connection, $method], $params);
                 $success = true;
-            } catch (\Exception $e) {
+            } catch (Exception $e) {
                 $lastException = $e;
-                \common_Logger::d('Redis  ' . $method . ' failed ' . $attempt . '/' . $retry . ' :  ' . $e->getMessage());
-                if ($e->getMessage() == 'Failed to AUTH connection' && isset($this->params['password'])) {
```

Comment:
This was existing logic for years, so I did not want to change the approach after the current small refactoring